### DISCUSSION TITLE
USB <-> RS485 adapter recommendation

### DISCUSSION THREAD

**Discussion by thorstenpawlak** [Louisvdw/dbus-serialbattery#114] [Q&A] [answered]
Can you recommend any adater cheaper than the Victron one? The Victron one is isolted right? Would definitley prefere that. Thanks

**Comment by pau1phi11ips** [comment]
The Victron one is indeed isolated. They also identify each USB connection uniquely in Venus. The cheapo USB -> RS485 adapters all have the same ID on the USB chip so you can only have 1 one of each adapter on the same system.

**Reply by merlep** [reply]
Victron report that their RS485-USB are not isolated , https://www.victronenergy.com/live/energy-meters:start - see Question #10.

**Comment by mr-manuel** [comment, marked as answer]
Check this site https://off-grid-garage.com/bms-communication/
